rustfmt.toml

```diff
@@ -8,7 +8,7 @@ imports_layout = "HorizontalVertical"
 imports_granularity = "Crate"
 match_block_trailing_comma = true
 max_width = 120
-newline_style = "Unix"
```

Comment:
I don't think this is correct.... we will get different formats on different machines